src/ui/components/Events/ReplayInfo.tsx

```diff
@@ -69,33 +71,49 @@ function ReplayInfo({ setModal }: PropsFromRedux) {
           ) : null}
         </div>
         <div className="group">
-          <Row>
-            <Icon
-              filename="external"
-              className="cursor-pointer bg-iconColor group-hover:bg-primaryAccent"
-            />
-            <div className="overflow-hidden overflow-ellipsis whitespace-pre" title={recording.url}>
-              <a href={recording.url} target="_blank" rel="noopener noreferrer">
-                {getDisplayedUrl(recording.url)}
-              </a>
-            </div>
-          </Row>
+          {recording.url != "" ? (
```

Comment:
I'd switch this to `recording.url ? ( … ) : null`

Reply:
Thanks, done!